src/julia-parser.scm

```diff
@@ -549,7 +549,9 @@
                            op))
                         (else '|.|)))))
 
-          ((opchar? c)  (read-operator port (read-char port)))
+          ((if (opchar? c)
+               (read-operator port (read-char port))
+               (accum-julia-symbol c port)))
```

Comment:
This prevents the following "invisible character" and "invalid character" errors from happening.